src/webui/www/private/scripts/contextmenu.js

```diff
@@ -267,6 +273,18 @@ window.qBittorrent.ContextMenu ??= (() => {
             return this;
         },
 
+        // disable an item
+        disableItem: function(item) {
+            this.menu.querySelector(`a[href$="${item}"]`).parentElement.classList.add("disabled");
```

Comment:
If I'm not mistaken, unless you use `:scope`, the `querySelector()` will search the full document.
I think we should start using `:scope` more.

Reply:
this.menu is menu element (ul) so querySelector only searches through its children and not entire document

Reply:
https://web.dev/articles/what-s-the-CSS-scope-pseudo-class-for#when_queryselector_goes_wrong